docs/integrations/github/discovery.md

```diff
@@ -76,31 +76,34 @@ catalog:
           repository: '.*' # optional Regex
 ```
 
-This provider supports multiple organizations via unique provider IDs
+This provider supports multiple organizations via unique provider IDs.
 
 > **Note:** It is possible but certainly not recommended to skip the provider ID level.
 > If you do so, `default` will be used as provider ID.
 
-- **catalogPath** _(optional)_:
```

Comment:
I meant to use **`catalogPath`** with code formatting like

```
**`catalogPath`**
```
instead of
```
**catalogPath**
```

I think for the documentation pages it is probably better if not the whole block is code formatted.

Reply:
Fixed. Thanks for clarifying.